### PR TITLE
240 transcript

### LINKED ISSUE #240: As an administrator, I want to be able to produce unofficial transcripts

### PR DESCRIPTION
**Pull request by oceancypress**
closes #240 
Test:
sign in as admin
go to students view
click on student row
select generate under transcript
criticize away! :P

### CODE REVIEW

File: student-info-system/test_transcript.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+from sis.models import Major
+from sis.tests.utils import createAdmin, createStudent
+
+
+class AdminTranscriptViewTest(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super(AdminTranscriptViewTest, cls).setUpTestData()
```
Comment (bjmckenz) [inline student-info-system/test_transcript.py:11]: Note that major now has a point of contact. Failing to include this made some tests fail. Just a heads up.

**Review by bjmckenz — APPROVED** [review]
Looks good

**Review by jeffreicher — COMMENTED** [review]
code review looks good, may want to update tests before merge

**Comment by jeffreicher** [comment]
error occurs when trying to click on a student in the students tab
image

**Comment by oceancypress** [comment]
> error occurs when trying to click on a student in the students tab
> image

This image shows that you are in the users tab, my changes are in the students tab. Please submit this as a bug in issues as it does not seem related to my branch @jeffreicher

**Pull request merged**